src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.ValidateToken.Internal.cs

```diff
@@ -150,7 +150,7 @@ await ValidateJWEAsync(jsonWebToken, validationParameters, currentConfiguration,
                 return result;
             }
 
-            if (TokenUtilities.IsRecoverableErrorType(result.UnwrapError().Type))
+            if (TokenUtilities.IsRecoverableException(result.UnwrapError().GetException()))
```

Comment:
Why are we allocating the exception when we could simply check the type and save the allocation?